samples/MvcSample.Web/Views/Shared/DisplayTemplates/Decimal.cshtml

```diff
@@ -4,8 +4,8 @@
 @functions {
     private object FormattedValue {
         get {
-            if (ViewData.TemplateInfo.FormattedModelValue == ViewData.ModelMetadata.Model) {
-                return String.Format(CultureInfo.CurrentCulture, "{0:0.00}", ViewData.ModelMetadata.Model);
+            if (ViewData.TemplateInfo.FormattedModelValue == ViewData.ModelExplorer.Model) {
```

Comment:
This was a big discussion yesterday and the decision was to stick with `ModelExplorer`. We will revisit during API reviews